src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -863,19 +864,16 @@ public ResponseEntity<String> getExportFormats() {
     @GetMapping(value = "/studies/{studyUuid}/root-networks/{rootNetworkUuid}/nodes/{nodeUuid}/export-network/{format}")
     @Operation(summary = "export the study's network in the given format")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The network in the given format")})
-    public ResponseEntity<byte[]> exportNetwork(
+    public void exportNetwork(
             @PathVariable("studyUuid") UUID studyUuid,
             @PathVariable("rootNetworkUuid") UUID rootNetworkUuid,
             @PathVariable("nodeUuid") UUID nodeUuid,
             @PathVariable("format") String format,
             @RequestParam(value = "formatParameters", required = false) String parametersJson,
-            @RequestParam(value = "fileName") String fileName) {
+            @RequestParam(value = "fileName") String fileName,
+            HttpServletResponse response) {
```

Comment:
for consistency? I don't know
```suggestion
            HttpServletResponse exportNetworkResponse) {
```